v1Tov2Migrations/migrate_approved_captures.js

```diff
@@ -14,8 +14,15 @@ async function migrate() {
         join field_data.wallet_registration wr on s.originating_wallet_registration_id = wr.id
         join public.trees pt on rc.id::text = pt.uuid
         left join treetracker.capture tc on rc.id = tc.id
-        where (rc.status = 'approved' and tc.id is null) or 
+        where 
+        (
+        (rc.status = 'approved' and tc.id is null) or 
         (rc.status != 'approved' and pt.active = true and pt.approved = true)
+        ) AND
+	pt.planting_organization_id = 1642
+	order by pt.id asc
+	--offset 0
+	limit 10000
```

Comment:
This is for temporary, because there are too many data, I migrate the new tree first.